Alignment SD-JWT Draft 22 and SD-JWT VC Draft 11 (#870)
italia/eidas-it-wallet-docs: docs/en/credential-data-model.rst

    * - **display**
      - REQUIRED. Array of objects, one for each language supported, containing display information for the Digital Credential type. It contains for each object the following properties:

>           * ``locale``: language tag as defined in Section 2 of :rfc:`5646`, the name of this parameter is aligned with SD-JWT-VC Draft 12. [REQUIRED].

Author: the "lang" parameter is renamed in SD-JWT VC Draft 12. This PR anticipates this change.

                    * ``uri#integrity``: integrity metadata as defined in Section 3 of `W3C-SRI`_. [REQUIRED].
                    * ``alt_text``: A string containing alternative text to display instead of the logo image. [OPTIONAL].

                * ``background_color``: RGB color value as defined in `W3C.CSS-COLOR`_ for the background of the Digital Credential. [OPTIONAL]. 

>                 * ``background_image``: Object containing information about the background image to be displayed for the type. This property is OPTIONAL [Aligned with SD-JWT-VC Draft 12]. The object contains the following sub-values:

Author: This PR anticipates this new paramater introduced in  SD-JWT VC Draft 12

          * ``display``: array containing display information about the claim indicated in the ``path``. The array contains an object for each language supported by the Digital Credential type. This property is REQUIRED. It contains the following members:
             * ``locale``: language tag as defined in Section 2 of :rfc:`5646`, the name of this parameter is aligned with SD-JWT-VC Draft 12. [REQUIRED].
             * ``label``: human-readable label for the claim. [REQUIRED].
             * ``description``: human-readable description for the claim. [REQUIRED].
          * ``sd``: string indicating whether the claim is selectively disclosable. It MUST be set to `always` if the claim is selectively disclosure or `never` if not. [REQUIRED].
          * ``svg_id``: alphanumeric string containing ID of the claim referenced in the SVG template as defined in [`SD-JWT-VC`_] Section 9. [REQUIRED].
      - [`SD-JWT-VC`_] Section 9.

A non-normative Digital Credential type metadata is provided below.

.. literalinclude:: ../../examples/vc-metadata-type.json
  :language: JSON

Digital Credential Type Metadata retrieval
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> In accordance with Section 6.3.3 of `SD-JWT-VC`_ the Credential Type Metadata JSON Document MAY be retrieved through a *well-known* endpoint. 

Author: ```suggestion
The Credential Type Metadata JSON Document MAY be retrieved through a *well-known* endpoint. See Section 6.3.3 of `SD-JWT-VC`_
```

          * ``path``: array indicating the claim or claims that are being addressed. [REQUIRED].
          * ``display``: array containing display information about the claim indicated in the ``path``. The array contains an object for each language supported by the Digital Credential type. This property is REQUIRED. It contains the following members:

>              * ``locale``: language tag as defined in Section 2 of :rfc:`5646`, the name of this parameter is aligned with SD-JWT-VC Draft 12. [REQUIRED].

Author: the "lang" parameter is renamed in SD-JWT VC Draft 12. This PR anticipates this change.
